### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sadam1195**
List of paper and references updated.

### CODE REVIEW

**Comment by r9y9** [comment]
Thank you very much!! 😊

File: README.md [hunk 1]
```diff
@@ -259,6 +259,7 @@ tensorboard --logdir=log
 - Parametric Resynthesis with neural vocoders https://arxiv.org/abs/1906.06762
 - Representation Mixing fo TTS Synthesis https://arxiv.org/abs/1811.07240
 - A Unified Neural Architecture for Instrumental Audio Tasks https://arxiv.org/abs/1903.00142
+- TACOTRON: TOWARDS END-TO-END SPEECH SYNTHESIS https://arxiv.org/pdf/1703.10135.pdf
```
Comment (r9y9) [inline README.md]: Oops, looking at this again, this seems incorrect. What was your intention?

Comment (Sadam1195) [reply README.md]: > Oops, looking at this again, this seems incorrect. What was your intention?

The address https://arxiv.org/pdf/1703.10135.pdf should be changed to https://arxiv.org/abs/1703.10135

Comment (r9y9) [reply README.md]: I meant the original Tacotron paper never used the repo in any way.

Comment (Sadam1195) [reply README.md]: > I meant the original Tacotron paper never used the repo in any way.

Oops, I meant to add that as reference reading material.

**Pull request merged**